rust/private/rustc.bzl

```diff
@@ -153,12 +153,11 @@ def collect_deps(label, deps, proc_macro_deps, aliases, toolchain):
                 ),
             )
 
-    # TODO: Fix depset union (https://docs.bazel.build/versions/master/skylark/depsets.html)
     direct_crates = []
-    transitive_crates = depset()
-    transitive_dylibs = depset(order = "topological")  # dylib link flag ordering matters.
```

Comment:
This strikes me as a notable omission, and I suspect that the breakage is not covered by tests (I vaguely recall encountering this in a relatively large build graph). Are you sure this is okay to drop?